src/components/List/ListItem.tsx

```diff
@@ -94,7 +108,22 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     accessibility: ListItemBehavior as Accessibility,
   }
 
-  state: any = {}
+  private itemRef = React.createRef<HTMLElement>()
+
+  private focusableItem = new FocusableItem(
+    () => this.props.focusableItemProps,
+    this.setState.bind(this),
+    state => {
+      this.state = { ...{ isHovering: false }, ...state }
```

Comment:
why do you need here this ```...{ isHovering: false }```?

It leads to one more ```Object.assign``` call, I think might be ok just ```this.state = { isHovering: false, ...state }```

Also, if ```...state``` will have ```isHovering:true``` it will override ```isHovering: false``` at the beginning.
Is it intended? if no, then better to swap them.